conf/log4j.properties

```diff
@@ -6,35 +6,50 @@
 # For the general syntax of property based configuration files see the
 # documenation of org.apache.log4j.PropertyConfigurator.
 
-log4j.threshhold=OFF
+status=error
+dest=err
+name=PropertiesConfig
 
-#log4j.rootLogger=INFO,A1
-log4j.logger.com.zimbra=INFO,A1
+rootLogger.level=INFO
+rootLogger.appenderRef.stdout.ref=ConsoleLogger
 
-log4j.logger.zimbra.qa.trace=INFO,A1
-log4j.logger.zimbra.qa.trace.result=INFO
+#log4j.rootLogger=INFO,A1
+#log4j.logger.com.zimbra=INFO,A1
+logger.zimbra.name=com.zimbra
+logger.zimbra.level=INFO, A1
+logger.zimbra.additivity=false
+logger.zimbra.appenderRef.A1.ref=ConsoleLogger
 
 # A1 is set to be a ConsoleAppender which outputs to System.out.
-log4j.appender.A1=org.apache.log4j.ConsoleAppender
-log4j.appender.A1.layout=org.apache.log4j.PatternLayout
-
+appender.A1.type=Console
+appender.A1.name=ConsoleLogger
+appender.A1.layout.type=PatternLayout
+appender.A1.layout.pattern=%d %-5p [%t] [%z] %c{1} - %m%n
 # The conversion pattern uses format specifiers. You might want to
 # change the pattern an watch the output format change.
-#log4j.appender.A1.layout.ConversionPattern=%-4r %-5p [%t] %c{1} - %m%n
-#log4j.appender.A1.layout.ConversionPattern=%-6r [%15.15t] %-5p %30.30c %x - %m\n
-#log4j.appender.A1.layout.ConversionPattern=%-4r %-5p [%t] %37c %3x - %m%n
+#appender.A1.layout.pattern=%-4r %-5p [%t] %c{1} - %m%n
+#appender.A1.layout.pattern=%-6r [%15.15t] %-5p %30.30c %x - %m\n
+#appender.A1.layout.pattern=%-4r %-5p [%t] %37c %3x - %m%n
 
 # Appender A2 writes to the file "test.log".
-log4j.appender.A2=org.apache.log4j.FileAppender
-log4j.appender.A2.File=StafTestCore.log
-
-log4j.appender.A2.Append=false
-log4j.appender.A2.layout=org.apache.log4j.PatternLayout
-#log4j.appender.A2.layout.ConversionPattern=%d %-5p [%t] %c{1} - %m%n
-#log4j.appender.A2.layout.ConversionPattern=%d %-5p [%t] %50c %3x - %m%n
-
+appender.A2.type=RollingFile
+appender.A2.name=StafTestCore
+appender.A2.fileName=StafTestCore.log
+appender.A2.layout.pattern=%d %-5p [%t] [%z] %c{1} - %m%n
+appender.A2.policies.type=Policies
+appender.A2.policies.time.type=TimeBasedTriggeringPolicy
+appender.A2.policies.time.interval=2
+appender.A2.policies.time.modulate=true
```

Comment:
Add a logger for A2 appender(StafTestCore)

Reply:
Yeah its there.